pkgs/top-level/all-packages.nix

```diff
@@ -39795,7 +39795,9 @@ with pkgs;
 
   picosat = callPackage ../applications/science/logic/picosat { };
 
-  libpoly = callPackage ../applications/science/logic/poly { };
+  pid1 = callPackage ../applications/virtualization/pid1 {};
+
+  libpoly = callPackage ../applications/science/logic/poly {};
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format so you don't have to edit this file

Reply:
Didn't know this and it is much nicer. Fixed, thanks!